big_tests/tests/metrics_register_SUITE.erl

```diff
@@ -102,6 +108,8 @@ unregister(Config) ->
     Alice = escalus_users:get_user_by_name(alice),
     escalus_users:delete_user(Config, Alice),
 
-    %% TODO replace sleep with backoff - wait until the metric is updated
-    timer:sleep(100),
-    assert_counter(Deregistarations + 1, modUnregisterCount).
+    factor_backoff(fun() -> assert_counter(Deregistarations + 1, modUnregisterCount) end,
+                   {value, Deregistarations + 1},
+                   30,
+                   10,
+                   150).
```

Comment:
Magic numbers.
Can we use `#{attempts => 10, sleep_time => 150, min_time => 666, max_time => 999}`? Or something like this.

`Fun` and `ExpectedValue` can remain as regular args.